### PR TITLE
week16

### PR DESCRIPTION
**Pull request by AmandaHockmuth**
Implemented mocha, chai, chai http, factory bot, faker and puppeteer for testing.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -68,7 +75,17 @@ const csrf_options = {
 };
 const csrf_middleware = csrf(csrf_options); //initialize and return middleware
```
Comment (jsuoz121) [inline app.js:76]: If you want I believe you can use app.use(csrf_middleware) to apply across all routes….but not essential!

File: test/test_multiply.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// const multiply = require("../util/multiply");
```
Comment (jsuoz121) [inline test/test_multiply.js:1]: I assume you verified these worked and then commented out?

**Review by jsuoz121 — APPROVED** [review]
Nice job as always! If you had any issues with the commented code lmk

File: test/test_multiply.js (hunk 2 above)
Comment (AmandaHockmuth) [reply test/test_multiply.js:1]: Yes I did. Once I got proof of concept I commented them out to clean up the console output

**Pull request merged**